src/coreclr/vm/codeman.cpp

```diff
@@ -4624,12 +4980,20 @@ BOOL ExecutionManager::IsManagedCodeWorker(PCODE currentPC, RangeSectionLockStat
     if (pRS == NULL)
```

Comment:
In the context of this function, currentPC isn't necessarily the eip register or equivalent, but instead a code address that may or may not be managed code, right? Or will we eventually need to handle 'currentPC is inside of the interpreter's native implementation, so we need to find the interpreter stack and fish out its IP'? I can't tell from grepping around whether that is potentially going to happen here.

(If it does matter for later, it'd just be cool to see a comment pointing out that problem or an interpreter-todo like the other ones you've added.)

Reply:
For the interpreter, PC always represents the interpreter PC, that means a byte code address pointer. That's the way everywhere. I see the interpreter byte code as an instruction set of a virtual processor.